modules/serverless-logic-supporting-services-eventing-system-config.adoc

```diff
@@ -0,0 +1,252 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-managing-supporting-services
+
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-logic-supporting-services-eventing-system-config_{context}"]
+= Supporting services eventing system configurations
+
+This section describes the supporting services eventing system configurations.
+
+In {ServerlessLogicProductName} installation, the following types of events are generated:
+
+* Outgoing and incoming events related to workflow business logic.
+
+* Events sent from workflows to the Data Index and Job Service.
+
+* Events sent from the Job Service to the Data Index Service.
+
+The {ServerlessLogicOperatorName} leverages the Knative Eventing system to manage all event communication between these services, ensuring efficient and reliable event handling.
+
+[id="serverless-logic-supporting-services-platform-scoped-eventing-system-config_{context}"]
+== Platform-scoped eventing system configuration
+
+To configure a platform-scoped eventing system, you can use the `spec.eventing.broker.ref` field in the `SonataFlowPlatform` CR to reference a Knative Eventing Broker. This configuration instructs the {ServerlessLogicOperatorName} to automatically link the supporting services to produce and consume events using the specified Broker.
+
+Additionally, workflows deployed in the same namespace with the `preview` or `gitops` profile, and without a custom eventing system configuration, will automatically link to the specified Broker.
+
+[NOTE]
```

Comment:
NOTE > IMPORTANT?

Seems important to me but I'm no expert!

Reply:
Changed it to "IMPORTANT"